src/macro_expansion.jl

```diff
@@ -124,21 +124,25 @@ function eval_macro_name(ctx::MacroExpansionContext, ex::SyntaxTree)
     ctx5, ex5 = linearize_ir(ctx4, ex4)
     mod = ctx.current_layer.mod
     expr_form = to_lowered_expr(mod, ex5)
-    eval(mod, expr_form)
+    try
+        eval(mod, expr_form)
+    catch
+        throw(MacroExpansionError(mctx, ex, "Error evaluating macro name", :all))
```

Comment:
The majority of the time this `eval()` will fail due to the macro not existing, so this is hopefully a simpler error for the end user

```suggestion
        throw(MacroExpansionError(mctx, ex, "Macro not found", :all))
```